app/assets/stylesheets/grader.scss

```diff
@@ -184,9 +184,8 @@
 
 .mark-deductions {
   clear: both;
-  color: $dark-blue;
-  cursor: pointer;
-  font-size: 1.1em;
+  color: $dark-grey;
```

Comment:
Hmmm, here can we just use the default value (which I'm assuming is `$black$`)?

Similarly, do we need to specify a `font_size` below, or can we use the default value there too?

Reply:
So I was basing the change for the default color on everything else on the page which is dark grey. When I remove the properties you describe, the default color is also $dark-grey, and the size is 0.8em/1.2em
<img width="328" alt="image" src="https://user-images.githubusercontent.com/29822176/89549991-0d577f80-d7d7-11ea-9804-b2e1bd786e0f.png">